AGENTS.md

```
@@ -0,0 +1,196 @@
+        ctx,
+        c.client,
+        recorder,
+        requiredDeployment,
+    )
+    return err
+}
+```
+
+## Key File Locations
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)